instagram/src/PostContainer/PostContainer.js

```diff
@@ -4,16 +4,23 @@ import CommentSection from '../CommentSection/CommentSection';
 import PropTypes from 'prop-types';
```

Comment:
i know your formatter does this, but having the opening and closing arrows split up like that, makes things harder to read. also i think at times, some environments will not be able to read it at all, if you can please, keep the arrows together, unless it is like an input field and it is self closing